packages/calcite-components/src/components/chip-group/chip-group.stories.ts

```diff
@@ -104,3 +104,36 @@ export const darkThemeRTL_TestOnly = (): string => html`
 `;
 
 darkThemeRTL_TestOnly.parameters = { themes: modesDarkDefault };
+
+export const multiPadding = (): string => html`
```

Comment:
Could we update an existing story to provide coverage for this instead of adding a padding-specific story?